### ISSUE TITLE
Show contacts on start up instead of Home screen once an account is configured

### ISSUE THREAD

**Issue by tbertels** [BelledonneCommunications/linphone-desktop#203]
The Invite your friends and especially the Account assistant are nice to have on the first start.
But once an account is configured and contacts have been added, the contacts screen should be shown on startup instead of that then useless screen:

![image](https://user-images.githubusercontent.com/3265870/43644062-a981c790-972d-11e8-8b41-634f8fc20776.png)

**Comment by WPFilmmaker** [comment]
maybe keep a small icon with "invite your friends"?

**Comment by shvchk** [comment]
Yeah, would be nice to have an option to show Contacts tab by default. Or Linphone might remember last used tab and use that by default.
